### ISSUE TITLE
[MAS2.4.6] [Headings and Labels -Help] Multiple h1 headings are defined for Get started with the bot inspector screen

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1809]
**Actual Result:**
Multiple h1 headings are defined for Get started with the bot inspector screen i.,e 
Below are h1 headings 
1.	Bot state inspection 
2.	Let’s get started 
3.	How it works 
4.	Run your bot in external channel and observing the bot framework emulator  

**Expected Result:**
H1 heading should be only one and multiple h1 headings should not define.

**Repro Steps:**
Turn on the NVDA 
1.Open the Bot Framework V4 Emulator.
2.Navigate to the help menu and select it 
3.Navigate to the get started with  channels(bot inspector) and select it  
4.get started with the bot inspector screen gets appears 
5.Navigate to the  headings mode using H key 
6.Navigate to the screen 
7.Observe the issue 

**User Impact:**
Users who rely on screen reader will be affected if multiple H1 headings are defined for screen then the users will be affected to navigate to the screen.  

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Screen reader: NVDA

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260}
[MAS2.4.6_Help.zip](https://github.com/microsoft/BotFramework-Emulator/files/3562268/MAS2.4.6_Help.zip)
